pipeline/steps/tf-plan.yaml

```diff
@@ -137,7 +139,31 @@ steps:
     inputs:
       command: 'plan'
       environmentServiceName: ${{ parameters.subscriptionName }}
-      commandOptions: '-var "common_tags={\"environment\":\"${{ parameters.env }}\",\"teamName\":\"$(product)\",\"BuiltFrom\":\"$(builtFrom)\"}" -var "location=$(location)" -var "product=$(product)" -var "env=${{ parameters.env }}" -var "cert_path=$(certPath)" -var "service_certificate_kv_url=$(secretId)" -var "thumbprint=$(thumbprint)" -var "key_vault_id=$(kvId)" -var "wowza_sku=$(wowza_sku)" -var "wowza_version=$(wowza_version)" -var "wowza_publisher=$(wowza_publisher)" -var "wowza_offer=$(wowza_offer)" -var "dns_zone_name=$(dns_zone_name)" -var "dns_resource_group=$(dns_resource_group)" -var "address_space=$(address_space)" -var "num_applications=$(num_applications)" -var "lb_IPaddress=$(lb_IPaddress)" -var "ssh_public_key=$(pubKey)" -var "rtmps_source_address_prefixes=[$(rtmps_source_address_prefixes)]" -var "ws_name=$(ws_name)" -var "ws_rg=$(ws_rg)" -var "ws_sub_id=$(ws_sub_id)" -out="${{ parameters.env }}.tfplan" -no-color -input=false'
+      commandOptions: >
+        -var "location=$(location)"
+        -var "product=$(product)"
+        -var "builtFrom=${{ parameters.builtFrom }}"
+        -var "environment=${{ parameters.env }}"
+        -var "env=${{ parameters.env }}"
+        -var "cert_path=$(certPath)"
+        -var "service_certificate_kv_url=$(secretId)"
+        -var "thumbprint=$(thumbprint)"
+        -var "key_vault_id=$(kvId)"
+        -var "wowza_sku=$(wowza_sku)"
```

Comment:
this is pretty crazy, thanks for re-formatting at least it's much more readable if still crazy having all these vars set here.